actors/evm/src/lib.rs

```diff
@@ -118,14 +119,23 @@ impl EvmContractActor {
 
         // TODO this does not return revert data yet, but it has correct semantics.
         if exec_status.reverted {
-            Err(ActorError::unchecked(EVM_CONTRACT_REVERTED, "constructor reverted".to_string()))
+            rt.exit(
```

Comment:
So, I personally _prefer_ exiting like this but... is there a good reason not to just return an error along with a value? Otherwise, this is the _only_ case.

Reply:
Actually, the trampoline doesnt even handle the output data from errors.

I should fix that, and _here_ we can just return the error if you like. I still think its more explicit if we call exit however.

Reply:
ok fixed the trampoline; i will rewrite this to use Errs so that it doesn't look odd.

Reply:
done.